frontend/src/Pages/StudentManagement.tsx

```diff
@@ -171,70 +173,79 @@ export default function StudentManagement() {
                         </button>
                     </div>
                 </div>
-                <table className="table">
+                <table className="table-2 w-full border-separate border-spacing-y-4">
                     <thead>
-                        <tr className="border-gray-600">
-                            <th className="flex">
-                                <span>Name</span>
+                        <tr>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Name
+                            </th>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Username
+                            </th>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Last Active
+                            </th>
+                            <th className="px-6 pb-2 text-left text-sm font-medium text-gray-1">
+                                Actions
                             </th>
-                            <th>Username</th>
-                            <th>Last Updated</th>
-                            <th>Actions</th>
                         </tr>
                     </thead>
                     <tbody>
                         {!isLoading &&
                             !error &&
-                            userData.map((user: User) => {
+                            userData.map((user) => {
                                 const updatedAt = new Date(user.updated_at);
                                 return (
                                     <tr
                                         key={user.id}
-                                        className="border-gray-600"
+                                        className="bg-white shadow-sm"
                                     >
-                                        <td>
-                                            {user.name_first} {user.name_last}
+                                        <td className="first:rounded-l-lg px-6 py-4">
+                                            <div className="text-sm text-gray-1">
+                                                {user.name_first}{' '}
+                                                {user.name_last}
+                                            </div>
+                                        </td>
+                                        <td className="px-6 py-4">
+                                            <div className="text-sm text-gray-1">
+                                                {user.username}
+                                            </div>
                                         </td>
-                                        <td>{user.username}</td>
-                                        <td>
+                                        <td className="px-6 py-4">
```

Comment:
Make sure these columns align with the headers! Right now they are slightly off.
<img width="1257" alt="Screenshot 2024-10-07 at 10 23 51 AM" src="https://github.com/user-attachments/assets/aefe301a-2e3d-409e-95bf-deb2b9d0b790">